pyam/utils.py

```diff
@@ -351,7 +354,10 @@ def convert_r_columns(c):
     # verify that there are no nan's left (in columns)
     null_rows = df.isnull().T.any()
     if null_rows.any():
-        raise_data_error("Empty cells in `data`", df.loc[null_rows])
+        cols = ", ".join(df.columns[df.isnull().any().values])
+        raise_data_error(
+            f"Empty cells in `data` (columns: '{cols}')", df.loc[null_rows]
```

Comment:
What would be very convenient is for columns that are called "unnamed: i" to convert it to the corresponding letter in the alphabet since that's how excel displays it. In the case of the test we have "unnamed: 7" -> counted from zero that corresponds to 8th letter, so we could write "unnamed: 7 (Excel column: H)" or something like that.
This way errors could be located even more quickly.

Reply:
This would be a nice improvement, but this validation happens after melting from wide to long format - therefore, there is no (simple) way to recover the line number.

Reply:
Ok, I had a hunch that that might be not so straightforward. Using the search function to find the faulty cells is totally fine too.